Podfile

```diff
@@ -404,7 +404,9 @@ pre_install do |installer|
   installer.pod_targets.each do |pod|
     # Statically linking Sentry results in a conflict with `NSDictionary.objectAtKeyPath`, but dynamically
     # linking it resolves this.
-    if pod.name == 'Sentry'
+    #
+    # The regex is to match both "Sentry" and "SentryPrivate" pods.
+    if pod.name =~ /^Sentry/
```

Comment:
@mokagio @crazytonyli Let me know if this change is okey. I couldn't update Sentry to 8.5.0 without this change. 

Apparently, Sentry 8.5.0 comes without another pod called `SentryPrivate` and it needs to be installed dynamically alongside the `Sentry` pod. Thus, the change.

Reply:
Nitpick: it's probably best to list out the specific pods if there are only two pods? i.e. `if %w[Sentry SentryPrivate].include? pod.name`?

Reply:
@crazytonyli Resolved in 450c777cf959d0d1e2d4b030c5eb4fa51281cf14